Octokit/Clients/RepositoryContentsClient.cs

```diff
@@ -36,6 +37,32 @@ public RepositoryContentsClient(IApiConnection apiConnection) : base(apiConnecti
         }
 
         /// <summary>
+        /// Returns the contents of a file or directory in a repository.
+        /// </summary>
+        /// <remarks>
+        /// If given a path to a single file, this method returns a collection containing only that file.
+        /// See the <a href="https://developer.github.com/v3/repos/contents/#get-contents">API documentation</a> for more information.
+        /// </remarks>
+        /// <param name="owner">The owner of the repository</param>
+        /// <param name="name">The name of the repository</param>
+        /// <param name="path">The content path</param>
+        /// <param name="reference">The name of the commit/branch/tag. Default: the repository�s default branch (usually master)</param>
+        /// <returns>
+        /// A collection of <see cref="RepositoryContent"/> representing the content at the specified path
+        /// </returns>
+        public async Task<IReadOnlyList<RepositoryContent>> GetContents(string owner, string name, string path, string reference)
```

Comment:
Because we maintain an set of `IObservable<T>` APIs that mirror this, the build server is currently complaining.

To fix this, you'll need to add the new method to `IObservableRepositoryContentsClient` and implement it over there too.